subprojects/core/src/test/groovy/org/gradle/api/internal/changedetection/state/CacheBackedTaskHistoryRepositoryTest.groovy

```diff
@@ -18,12 +18,10 @@ package org.gradle.api.internal.changedetection.state
 
 import org.gradle.api.UncheckedIOException
 import org.gradle.api.internal.changedetection.rules.AbstractTaskStateChangesTest
-import org.gradle.api.internal.tasks.GenericFileNormalizer
+import org.gradle.internal.fingerprint.AbsolutePathInputNormalizer
 import org.gradle.normalization.internal.InputNormalizationStrategy
 import spock.lang.Issue
 
-import static org.gradle.api.internal.changedetection.state.PathNormalizationStrategy.ABSOLUTE
-
 class CacheBackedTaskHistoryRepositoryTest extends AbstractTaskStateChangesTest {
     static final NORMALIZATION_STRATEGY = InputNormalizationStrategy.NOT_CONFIGURED
```

Comment:
I think we should find a better name for this, like `NOT_CONFIGURABLE`.

Reply:
How about `NO_NORMALIZATION` or `NOTHING_CONFIGURED`?

Reply:
`NO_NORMALIZATION` sounds best I think.

Reply:
`NO_NORMALIZATION` it is.